packages/libbinaryen/libbinaryen.106.0.0/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+synopsis: "Libbinaryen packaged for OCaml"
+maintainer: "blaine@grain-lang.org"
+authors: "Blaine Bublitz"
+license: "Apache-2.0"
+homepage: "https://github.com/grain-lang/libbinaryen"
+bug-reports: "https://github.com/grain-lang/libbinaryen/issues"
+depends: [
+  "conf-cmake" {build}
+  "dune" {>= "2.9.1" & < "3.0.0"}
```

Comment:
Why such a strict bound is needed?

Reply:
Dune 3 has an implicit dependency on opam on the system. In places you don't have opam, it fails. I want to ensure our package works in different contexts without needing to know that trivia.

Dune merged some patches this week that are yet-to-be-released. Once it works without opam, I'll open up the range.